device/celestica/x86_64-cel_silverstone-r0/Silverstone/port_config.ini

```diff
@@ -0,0 +1,35 @@
+# name          lanes                                   alias         index
+Ethernet0       33,34,35,36,37,38,39,40                  QSFPDD1       1
+Ethernet4       41,42,43,44,45,46,47,48                  QSFPDD2       2
+Ethernet8       49,50,51,52,53,54,55,56                  QSFPDD3       3
+Ethernet12      57,58,59,60,61,62,63,64                  QSFPDD4       4
+Ethernet16      65,66,67,68,69,70,71,72                  QSFPDD5       5
+Ethernet20      73,74,75,76,77,78,79,80                  QSFPDD6       6
+Ethernet24      81,82,83,84,85,86,87,88                  QSFPDD7       7
+Ethernet28      89,90,91,92,93,94,95,96                  QSFPDD8       8
+Ethernet32      1,2,3,4,5,6,7,8                          QSFPDD9       9
+Ethernet36      9,10,11,12,13,14,15,16                   QSFPDD10      10
+Ethernet40      17,18,19,20,21,22,23,24                  QSFPDD11      11
+Ethernet44      25,26,27,28,29,30,31,32                  QSFPDD12      12
+Ethernet48      97,98,99,100,101,102,103,104             QSFPDD13      13
+Ethernet52      105,106,107,108,109,110,111,112          QSFPDD14      14
+Ethernet56      113,114,115,116,117,118,119,120          QSFPDD15      15
+Ethernet60      121,122,123,124,125,126,127,128          QSFPDD16      16
+Ethernet64      129,130,131,132,133,134,135,136          QSFPDD17      17
+Ethernet68      137,138,139,140,141,142,143,144          QSFPDD18      18
+Ethernet72      145,146,147,148,149,150,151,152          QSFPDD19      19
+Ethernet76      153,154,155,156,157,158,159,160          QSFPDD20      20
+Ethernet80      225,226,227,228,229,230,231,232          QSFPDD21      21
+Ethernet84      233,234,235,236,237,238,239,240          QSFPDD22      22
+Ethernet88      241,242,243,244,245,246,247,248          QSFPDD23      23
+Ethernet92      249,250,251,252,253,254,255,256          QSFPDD24      24
+Ethernet96      161,162,163,164,165,166,167,168          QSFPDD25      25
+Ethernet100     169,170,171,172,173,174,175,176          QSFPDD26      26
+Ethernet104     177,178,179,180,181,182,183,184          QSFPDD27      27
+Ethernet108     185,186,187,188,189,190,191,192          QSFPDD28      28
+Ethernet112     193,194,195,196,197,198,199,200          QSFPDD29      29
+Ethernet116     201,202,203,204,205,206,207,208          QSFPDD30      30
+Ethernet120     209,210,211,212,213,214,215,216          QSFPDD31      31
+Ethernet124     217,218,219,220,221,222,223,224          QSFPDD32      32
+Ethernet125     257                                      SFP1          33
```

Comment:
Just a note here, we need  BRCM SAI fixes to have the SFP ports supported, otherwise, we would get crashes here due to SAI returning wrong lanes.  Anyway, I know SAI fix will be publish soon, so we should be OK.

Reply:
We will remove it firstly and validate 32x400 configuration.

Reply:
> We will remove it firstly and validate 32x400 configuration.

10G config is commented out at 20a2fed.